src/UrlParams.ts

```diff
@@ -235,6 +263,17 @@ class ParamParser {
     return this.fragmentParams.get(name) ?? this.queryParams.get(name);
   }
 
+  public getEnumParam<T extends string>(
+    name: string,
+    type: { [s: string]: T } | ArrayLike<T>,
+  ): T | undefined {
+    const value = this.getParam(name);
+    if (value && Object.values(type).includes(value as T)) {
```

Comment:
```suggestion
    if (value !== null && Object.values(type).includes(value as T)) {
```

To avoid surprises with falsy strings